userguide/tutorials/aws-container.adoc

```diff
@@ -24,29 +27,40 @@ image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-
 
 If all goes well, you should see a success screen:
```

Comment:
Looks like the success screen image is missing.

Reply:
Added the image reference